src/OpenIDConnectClient.php

```diff
@@ -812,6 +812,9 @@ protected function requestTokens($code) {
      */
     public function refreshToken($refresh_token) {
         $token_endpoint = $this->getProviderConfigValue('token_endpoint');
+        $token_endpoint_auth_methods_supported = $this->getProviderConfigValue('token_endpoint_auth_methods_supported', ['client_secret_basic', 'client_secret_post']);
```

Comment:
[According to the spec](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication), the default should be `client_secret_basic` only.